yarn.lock

```diff
@@ -1922,7 +1922,32 @@ babel-code-frame@^6.26.0:
     esutils "^2.0.2"
     js-tokens "^3.0.2"
 
-babel-core@^6.26.0, babel-core@^7.0.0-bridge.0:
+babel-core@^6.26.0:
```

Comment:
Seems like the `yarn.lock` is broken. Have you maybe forgotten to run `yarn` in the root? 😅